SPECS/glibc/glibc.spec

```diff
@@ -288,14 +295,23 @@ grep "^FAIL: nptl/tst-eintr1" tests.sum >/dev/null && n=$((n+1)) ||:
 %defattr(-,root,root)
 # TODO: Excluding for now to remove dependency on PERL
 # /usr/bin/mtrace
-%{_lib64dir}/*.a
-%{_lib64dir}/*.o
+%{_lib64dir}/{g,M,S}crt1.o
+%{_lib64dir}/crti.o
+%{_lib64dir}/crtn.o
```

Comment:
Why are we leaving some `.o` files here and some are moved to the new subpackage?

Reply:
I've kept the CRT files needed for linking dynamic executables (and PIEs) and shared objects in `glibc-devel` and moved the ones for `static` and `static-pie` binaries to the `glibc-static` package.

The goal is to force packages that statically link to `glibc` to explicitly depend on `glibc-static` (today they can depend on `glibc-devel` and link statically - or even don't depend on `glibc` at all and still statically link to it because it's in `pkggen`)